usr/share/rear/build/default/985_fix_broken_links.sh

```diff
@@ -0,0 +1,11 @@
+# build/default/985_fix_broken_links.sh
+# Check for broken symbolic links in our ROOTFS_DIR area - issue #1638
+
+pushd $ROOTFS_DIR >/dev/null
+    find . -xtype l | grep -v "/dev/" | while read SYMLINK
+    do
+        mising_file=$( ls -l $SYMLINK | awk '{print $11}' | sed -e 's/\.//g' )
```

Comment:
This sed looks wrong. A relative symlink like `../efi/EFI/redhat/grubenv
` will get stripped of all the dots and be converted to `/efi/EFI/redhat/grubenv`. The directory `/efi/EFI/redhat` will then get included in the rootfs (actually observed) and the desired file will not be copied, as the arguments to cp will be wrong:
`cp -v /efi/EFI/redhat/grubenv .//efi/EFI/redhat/grubenv`
(Also `readlink` would be more elegant than `ls` and `awk`, if one can afford to use it, but that's not the big issue)

Reply:
@pcahyna 
I think you are right.
I think the current code does not work sufficiently fail safe.
I think the current code does not cover all possible cases.
I think to get symlink targets into the recovery system
code as in build/GNU/Linux/390_copy_binaries_libraries.sh
when `$lib is a symbolic link on the original system` is also needed here.